docker/env/version

```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
 1.60-update-azure-compute-for-branch-2024.1
```

Comment:
@fruch is it typo, or we used 2024.1 marked hydra image for 2023.1? If we really used, I could also reuse one from 2024.1 which I just created

Reply:
> @fruch is it typo, or we used 2024.1 marked hydra image for 2023.1? If we really used, I could also reuse one from 2024.1 which I just created

It's not a typo, those branches were identical at some point, you can reuse the image